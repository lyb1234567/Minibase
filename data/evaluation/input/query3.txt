Q(x,y) :- R(x, y, z), x < 10 ,  y > 3
