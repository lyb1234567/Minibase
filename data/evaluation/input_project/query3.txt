Q(z,x) :- R(x, y, z)
