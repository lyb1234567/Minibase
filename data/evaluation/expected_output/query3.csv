1, 9, 'adbs'
2, 7, 'anlp'
8, 9, 'rl'
8, 9, 'ppls'
